src/coreclr/gc/gcpriv.h

```diff
@@ -184,6 +184,11 @@ inline void FATAL_GC_ERROR()
 
 #define BACKGROUND_GC   //concurrent background GC (requires WRITE_WATCH)
 
+// It is no longer possible to build without BACKGROUND_GC defined, so we have to use this duplicate
```

Comment:
@cshung You have been fixing the build without `BACKGROUND_GC` defined in the past: https://github.com/dotnet/runtime/pull/77328 . Is building without `BACKGROUND_GC` defined still something you care about?

Reply:
https://github.com/dotnet/runtime/pull/91445 should fix the build with undefined `BACKGROUND_GC`